coffee_cmd/src/main.rs

```diff
@@ -58,11 +58,6 @@ async fn main() -> Result<(), CoffeeError> {
             coffee_term::show_list(remotes)
         }
         CoffeeCommand::Upgrade { repo, verbose } => {
-            let spinner = if !verbose {
```

Comment:
this commit can be drop, `git rebase -i HEAD~5` can help you in here, do this experimentation in another copy of the branch because you can make irreversible mistake

Reply:
done